test/basic.html

```diff
@@ -51,6 +51,10 @@
             assert.equal(pages.selected, undefined);
           });
 
+          test('null activateEvent', function() {
```

Comment:
I didn't write the test suite, so I won't be able to defend it adequately. Here is my rationale for proposing this change:

There is a fundamental bug in `iron-pages`: for a specific user interaction, an unexpected interface change occurred. In order to cover regressions, a test must concern itself with the user interaction that `activeEvent`'s state interfered with, not just the state of `activeEvent` in isolation.

When `IronSelectableBehavior`'s related API eventually changes, this test will fail. An unsuspecting maintainer may look upon this test for the first time, hoping to fix the issue, and have little context to help her understand why it is important.

A comment (like the one you made in the source file) would be helpful. A test that addresses the potential regression would be good too, but perhaps not worth the effort.